x-pack/solutions/observability/plugins/streams_app/public/components/stream_detail_routing/current_stream_entry.tsx

```diff
@@ -32,14 +40,36 @@ export function CurrentStreamEntry({ definition }: { definition: WiredStreamGetR
     }
   );
 
+  const { euiTheme } = useEuiTheme();
+
   return (
     <>
       {!isRoot(definition.stream.name) && (
-        <EuiBreadcrumbs breadcrumbs={breadcrumbs} truncate={false} />
+        <EuiBreadcrumbs
+          max={2}
+          breadcrumbs={breadcrumbs}
+          truncate={false}
+          css={css`
+            padding-bottom: 8px;
+          `}
+        />
       )}
       <EuiFlexItem grow={false}>
-        <EuiPanel hasShadow={false} hasBorder paddingSize="s">
-          <EuiText size="s">{definition.stream.name}</EuiText>
+        <EuiPanel
+          hasShadow={false}
+          hasBorder
+          css={css`
+            padding: 8px 12px;
```

Comment:
Don't use arbitrary values, please get them from the team such `padding: ${euiTheme.size.s} ${euiTheme.size.m}`